src/content/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations.mdx

```diff
@@ -105,6 +105,8 @@ To add your Azure app to New Relic:
 2. Follow the steps in the UI to activate the integration in New Relic.
 3. If you have already completed the Azure account steps, skip to the end of the steps to fill out the form. (For `Azure account name`, enter the name you want to use to identify the account in your **Integrations** dashboard.)
 
+After you activate an Azure integration, we'll start monitoring your Azure data at regular [polling intervals](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-polling-intervals-infrastructure-integrations). To [find and use your data](/docs/infrastructure/integrations/find-use-infrastructure-integration-data), use the [data explorer](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer) or go to [**one.newrelic.com**](https://one.newrelic.com) **> Infrastructure > Azure**.
```

Comment:
Looks like there is extra ** in the path info

Reply:
Correct!